cogs5e/sheets/gsheet.py

```diff
@@ -360,7 +360,7 @@ def get_skills_and_saves(self):
 
         skills = {}
         saves = {}
-        is_joat = self.version == 2 and bool(character.value("AR45"))
+        is_joat = self.version == 2 and bool(character.value("AR45")) or self.version == 2.1 and bool(character.value("AQ59"))
```

Comment:
I feel like this needs some parentheses for each statement on either side of the `or`